packages/rpc/src/types.ts

```diff
@@ -31,6 +32,7 @@ export type CfSupportedAssetsResponse = RpcResponse<'cf_supported_assets'>;
 export type CfSwappingEnvironmentResponse = RpcResponse<'cf_swapping_environment'>;
 export type CfSwapRateResponse = RpcResponse<'cf_swap_rate'>;
 export type CfSwapRateV2Response = RpcResponse<'cf_swap_rate_v2'>;
+// export type CfPoolDepthResponse = RpcResponse<'cf_pool_depth'>;
```

Comment:
can't remember why i added these but i'm sure they're useful
```suggestion
export type CfPoolDepthResponse = RpcResponse<'cf_pool_depth'>;
```